lib/posix-process/exportsyms.uk

```diff
@@ -73,6 +73,7 @@ uk_syscall_e_gettid
 exit
 uk_syscall_r_exit
 uk_syscall_e_exit
+uk_sys_exit_group
```

Comment:
I think we don't have a use for exporting `exit_group()` in the kernel API. I also had a look at the [patch](https://github.com/unikraft/unikraft/pull/1364/commits/a4d929510ab7035c3871c3c715420fcb32cfd9b2) in the graceful shutdown PR that implements `exit()` / `exit_group()` and confirmed we only use such a function internally in `libposixprocess`. So maybe we can make `uk_sys_exit_group()` a `static` instead to avoid making a syscall from the kernel, and since it's no longer part of the kernel API, to rename it to something else (`pprocess_exit()` is okay as I'm using that in #1364).